index.html

```diff
@@ -0,0 +1,452 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script type="text/javascript" src="javascript/main.js" defer></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <div class="cancel">&times;</div>
+        <ul>
+          <li class="nav_link">
+            <a class="nav-link" href="#projects">Portoflio</a>
+          </li>
+          <li class="nav_link">
+            <a class="nav-link" href="#about-me">About</a>
+          </li>
+          <li class="nav_link">
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Make sure to match your design like in Figma, please change your design to match the colors, the horizontal line below the mobile menu item, and font weight.

Your design              | Figma template design
---------------------- | ------------------------------
![1](https://user-images.githubusercontent.com/53402600/169387336-879995b4-5ded-4649-b13a-3fc8e81d1bc9.png) | ![figma design](https://user-images.githubusercontent.com/53402600/169387393-46e3c356-dc1b-4d81-8868-de1a441defb5.png)